cookbook/bundles/best_practices.rst

```diff
@@ -221,7 +218,7 @@ following standardized instructions in your ``README.md`` file.
     following command to download the latest stable version of this bundle:
 
     ```bash
-    $ composer require <package-name> "~1"
+    $ composer require <package-name>
```

Comment:
please, do not remove this version number. If you have documentation for 1.0 version of your docs, it should explain installing version 1. If you release version 2.0 of your library, the 1.0 version of your docs should still explain the version 1 installation. Removing the version number means the 1.0 version of the docs makes you install 2.0